terraform/test_cluster/secrets.tf

```diff
@@ -17,9 +18,9 @@ resource "kubernetes_secret" "staging-secrets" {
   }
 
   data = {
-    "DATABASE_URL"                 = var.staging_database_url
-    "FUSIONAUTH_API_KEY"           = var.staging_fusionauth_api_key
-    "LEGACY_BACKEND_SHARED_SECRET" = var.staging_legacy_backend_shared_secret
-    "MAILCHIMP_API_KEY"            = var.mailchimp_api_key
+    "DATABASE_URL"                    = var.staging_database_url
+    "FUSIONAUTH_API_KEY"              = var.staging_fusionauth_api_key
+    "LEGACY_BACKEND_SHARED_SECRET"    = var.staging_legacy_backend_shared_secret
+    "MAILCHIMP_TRANSACTIONAL_API_KEY" = var.mailchimp_transactional_api_key
```

Comment:
Elsewhere we have both the mailchimp API key and the transactional API key - should we have both here too?

Reply:
Yes, we should, didn't realize the marketing key wasn't there yet

Reply:
Oh, it was there and I accidentally replaced it. Good catch!